test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -36,17 +39,32 @@ auto mk_local_peer(uint64_t num) {
 class FixedCryptoProvider : public MockYacCryptoProvider {
  public:
   explicit FixedCryptoProvider(const std::string &public_key) {
-    pubkey.fill(0);
-    std::copy(public_key.begin(), public_key.end(), pubkey.begin());
+    auto tmp =
+        shared_model::crypto::DefaultCryptoAlgorithmType::generateKeypair()
```

Comment:
Add todo for fixing in with `generateKeypair` interface from the adapter.

Reply:
IR-1189